### PR TITLE
Fix merge errors with #1060 and #1062

### PR DESCRIPTION
**Pull request by daniel-thom**
The PRs #1060 and #1062 were logically incompatible but didn’t have any git merge conflicts. This resolves the errors.

### CODE REVIEW

File: test/test_simulation_results.jl [hunk 1]
```diff
@@ -864,21 +891,6 @@ function test_simulation_results_from_file(path::AbstractString, c_sys5_hy_ed, c
 
     @test_throws IS.InvalidValue set_system!(results_uc, c_sys5_hy_ed)
 
-    current_file = joinpath(
```
Comment (daniel-thom) [inline test/test_simulation_results.jl:867]: The system will always be available in the simulation_store.h5 and so this condition should be impossible.

**Review by jd-lara — APPROVED** [review]

File: test/test_simulation_results.jl (hunk 1 above)
Comment (GabrielKS) [reply test/test_simulation_results.jl:867]: Yeah, makes sense — @tengis-nrl sorry for making you write this test

**Review by GabrielKS — APPROVED** [review]
Looks good to me!

File: test/test_simulation_results.jl (hunk 1 above)
Comment (tengis-nrl) [reply test/test_simulation_results.jl:867]: It did allow me to learn more about system files. No problem there.

**Pull request merged**